Signal/src/ViewControllers/OWSConversationSettingsTableViewController.m

```diff
@@ -240,16 +240,12 @@ - (void)updateTableContents
     firstSection.customHeaderHeight = @(100.f);
 
     if (!self.isGroupThread && self.thread.hasSafetyNumbers) {
-        NSString *recipientId = self.thread.contactIdentifier;
-        BOOL isVerified = [[OWSIdentityManager sharedManager] verificationStateForRecipientId:recipientId]
-            == OWSVerificationStateVerified;
-
         [firstSection addItem:[OWSTableItem itemWithCustomCellBlock:^{
             return [weakSelf
                 disclosureCellWithName:
                     NSLocalizedString(@"VERIFY_PRIVACY",
                         @"Label for button or row which allows users to verify the safety number of another user.")
-                              iconName:(isVerified ? @"table_ic_verify" : @"table_ic_not_verified")];
+                              iconName:@"table_ic_not_verified"];
```

Comment:
ah, good, so per my earlier comment, looks like we're not changing this icon depending on verification state. 

nit: the icon filename is weird.